bookDetails.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```

Comment:
I dont know why each pr it shows you added this, is this necessary or could be deleted?

Reply:
This is because I renamed the file, before this was the index